packages/experimental/src/global/js/settings.js

```diff
@@ -3,5 +3,13 @@ import { settings } from 'carbon-components';
 
 const pkgPrefix = 'exp';
 const carbonPrefix = settings.prefix;
+const storybookPrefixCanary = 'Cloud & Cognitive/Canary';
+const storybookPrefixReleased = 'Cloud & Cognitive/Released';
```

Comment:
As these are a Storybook construct, these ought to live in `packages/cloud-cognitive/config.js` or `packages/experimental/config.js`, similar to the convention of other packages, otherwise, they'll be included as part of the distributable.

Reply:
Moved to config